internal/pkg/scorecard/olm_tests.go

```diff
@@ -254,17 +256,13 @@ func (t *CRDsHaveResourcesTest) Run(ctx context.Context) *TestResult {
 					}
 				}
 				if foundResource == false {
-					allResourcesListed = false
 					missingResources = append(missingResources, fmt.Sprintf("%s/%s", resource.Kind, resource.Version))
 				}
 			}
-			if allResourcesListed {
-				res.EarnedPoints++
-			}
 		}
 	}
-	if res.EarnedPoints < res.MaximumPoints {
-		res.Suggestions = append(res.Suggestions, fmt.Sprintf("Add resources to owned CRDs for %s: %v", t.CR.GroupVersionKind(), missingResources))
+	if len(missingResources) > 0 {
+		res.Suggestions = append(res.Suggestions, fmt.Sprintf("Detected used resources not listed in resources section for owned CRD %s: %v", t.CR.GroupVersionKind(), missingResources))
```

Comment:
I feel like the phrasing for this line is a bit odd for a suggestion, but I wasn't sure what to change this to. Any suggestions to make this clearer?

Example of current output:
```
SUGGESTION: Detected used resources not listed in resources section for owned CRD cache.example.com/v1alpha1, Kind=MemcachedRS: [pods/v1 configmaps/v1]
```

Reply:
Maybe something like:

```
SUGGESTION: If it would be helpful to an end-user to understand or troubleshoot your CR, consider adding resources (pods/v1, configmaps/v1) to the resources section for owned CRD MemcachedRS.
```

Would it be helpful to somehow link to this doc if there are any errors or suggestions that result from the OLM test? 

https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md

Reply:
Since we're moving to a more generic scorecard where the Basic and OLM tests will be plugins, I don't think we can link based on if the OLM test has errors or suggestions. We could add that as a separate suggestion that the tests themselves can add if they fail though.